modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -213,7 +222,11 @@ const EditorUpload = (editor: Editor): EditorUpload => {
         if (imagesToRemove.length > 0 && !Rtc.isRtc(editor)) {
           editor.undoManager.transact(() => {
             Arr.each(imagesToRemove, (element) => {
-              editor.dom.remove(element);
+              const sugarElement = SugarElement.fromDom(element);
```

Comment:
You can use SugarElements.fromDom here that and pass that into Arr.each then the element will be sugar. Then use `element.dom.src` not Attr.get since if I recall src is always the full path while attributes is the value in the dom.